src/Microsoft.Framework.PackageManager/Sources/ListSourcesCommand.cs

```diff
@@ -0,0 +1,41 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Linq;
+
+namespace Microsoft.Framework.PackageManager
+{
+    internal class ListSourcesCommand
+    {
+        public Reports Reports { get; set; }
+        public string TargetDirectory { get; set; }
+
+        public int Execute()
+        {
+            // Most of this code is from NuGet, just some refactoring for our system.
+            // Collect config
+            var config = NuGetConfig.ForSolution(TargetDirectory);
+
+            var sources = config.Sources.LoadPackageSources().ToList();
+            if (!sources.Any())
+            {
+                Reports.Information.WriteLine("No sources found.");
+            }
+            else
+            {
+                Reports.Information.WriteLine("Registered Sources:");
+                // Iterate over the sources and report them
+                var nameLength = sources.Max(s => s.Name.Length);
+                for (int i = 0; i < sources.Count; i++)
+                {
+                    var source = sources[i];
+                    var indexNumber = i + 1;
+                    var enabledString = source.IsEnabled ? "" : " [Disabled]";
+                    var line = $"{indexNumber.ToString("0\\.").PadRight(3)} {source.Name.PadRight(nameLength)} {source.Source}{enabledString.Yellow().Bold()}";
```

Comment:
I'm mostly trying to mirror the existing `nuget sources` command, but I'm open to changing it. I'll remove the table formatting for now, we can always tweak it later.